queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon'; -- Find all animals whose name ends in "mon".
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01'; -- List the name of all animals born between 2016 and 2019.
```

Comment:
- [ ] This query is meant to List the names of all animals born between 2016 and 2019, So your query is supposed to start from the beginning of 2016 to the end of 2019, don't the beginning of 2019 as you did. Please fix that